### PR TITLE
chore: add github issues templates

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [X] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Add Github issue templates to have single view issues

### CODE REVIEW

**Review by jmickey — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: "🙏 Feature Request"
+about: Submit a feature request
+title: "[🙏]"
+labels: "🙏 feature request"
+assignees: ''
+---
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:7]: I don't quite understand this table, how is it meant to be used?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: "🐛 Bug report"
+about: Submit a bug
+title: "[🐛]"
+labels: "🐛  bug"
+assignees: ''
+---
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:7]: I don't quite understand this table, how is it meant to be used?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- `kubernetes-monitor version [e.g. v1.53.0]` 
+- `Cloud runtime [e.g. GKE, EKS]`
+
+### Expected behaviour
+Please share _expected_ behaviour.
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:13]:
```suggestion
Please share the _expected_ behaviour.
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+### Actual behaviour
+Please share _problematic_ behaviour you are seeing.
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:16]:
```suggestion
Please share the _problematic_ behaviour you are seeing.
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+### Steps to reproduce
+Please share _minimal_ steps needed to reproduce your issue. Ideally
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:19]:
```suggestion
Please share the _minimal_ steps needed to reproduce your issue. Ideally
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+a paired down manifest / project to showcase the problem that can also
```
Comment (ivanstanev) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:20]:
```suggestion
a container image or a YAML file to showcase the problem that can also
```

**Review by ivanstanev — COMMENTED** [review]
Nice one!

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md (hunk 1 above)
Comment (dkontorovskyy) [reply .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:7]: Apparently, it transforms into this table with requirements to the Issue look
https://github.com/snyk/snyk/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md (hunk 2 above)
Comment (dkontorovskyy) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md:7]: Apparently, it transforms into this table with requirements to the Issue look
https://github.com/snyk/snyk/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

**Pull request merged**